redbot/cogs/audio/core/commands/miscellaneous.py

```diff
@@ -45,21 +45,22 @@ async def command_sing(self, ctx: commands.Context):
     async def command_audiostats(self, ctx: commands.Context):
         """Audio stats."""
         server_num = len(lavalink.active_players())
-        total_num = len(lavalink.all_players())
+        total_num = len(lavalink.all_connected_players())
 
         msg = ""
-        async for p in AsyncIter(lavalink.all_players()):
-            connect_start = p.fetch("connect")
+        async for p in AsyncIter(lavalink.all_connected_players()):
             connect_dur = self.get_time_string(
-                int((datetime.datetime.utcnow() - connect_start).total_seconds())
+                int(
+                    (datetime.datetime.now(datetime.timezone.utc) - p.connected_at).total_seconds()
+                )
             )
```

Comment:
Perhaps we could put a default here in case not a full second passed yet?
```suggestion
            ) or "0s"
```
![image](https://user-images.githubusercontent.com/6032823/118842903-dd8ee980-b8c9-11eb-9bbc-53a708400df7.png)

Optional cause it's not something that this PR causes.

Reply:
:aha: how fast did you run the command lmfao